docs/diagrams/LessonClassDiagram.puml

```diff
@@ -0,0 +1,22 @@
+@startuml
+!include style.puml
+skinparam arrowThickness 1.1
+skinparam arrowColor MODEL_COLOR
+skinparam classBackgroundColor MODEL_COLOR
+
+
+Class Lesson
+Class LessonName
+Class Capacity
+Class Price
+Class Timing
+Class StudentName
+
+Lesson *--> "1"LessonName
+Lesson *--> "0..1"Capacity
+Lesson *--> "0..1"Price
+Lesson *--> "0..1"Timing
```

Comment:
Should these multiplicities be `1` instead, since they are guaranteed to be non-null at all times?